src/node_debugger.cc

```
@@ -188,6 +213,14 @@ class NodeDebuggerContext {
   static void DisconnectFromInspectorBackend(const v8::FunctionCallbackInfo<v8::Value>& info) {
     NodeDebugger::instance()->Disconnect();
   }
+
+  static void WaitForStartup(const v8::FunctionCallbackInfo<v8::Value>& info) {
+    if (!info[0]->IsFunction()) {
+      fprintf(stderr, "FAILURE, STARTUP CB NOT FUNCTION");
```

Better to throw an exception here?